core/src/main/java/com/sequenceiq/cloudbreak/service/environment/EnvironmentClientService.java

```diff
@@ -97,6 +101,16 @@ public String getCrnByName(String environmentName) {
         }
     }
 
+    public DetailedEnvironmentResponse getByCrnAsInternal(String environmentCrn) {
+        DetailedEnvironmentResponse environment = null;
+        if (Objects.nonNull(environmentCrn)) {
+            environment = measure(() -> ThreadBasedUserCrnProvider.doAsInternalActor(() -> getByCrn(environmentCrn)),
+                    LOGGER,
+                    "Get Environment from Environment service in StackToCloudStackConverter took {} ms");
```

Comment:
Nit: replace `StackToCloudStackConverter` with `EnvironmentClientService` (or simply remove the class name altogether).